pylsp_black/plugin.py

```diff
@@ -41,6 +41,22 @@ def pylsp_format_range(config, document, range):
     return format_document(config, document, range)
 
 
+@hookimpl
+def pylsp_settings():
+    """Configuration options that can be set on the client."""
+    return {
+        "plugins": {
+            "black": {
+                # Disable this plugin by default because it's third-party.
+                "enabled": False,
```

Comment:
I think this is going to break most users' installations when people upgrade. Would it be too bad if we enables it by default and documented how to disable it?

Reply:
My rationale is that if the plugin is installed that means it's intended to be used, so having to set the configuration is another step in the way.

Reply:
Ok, no problem. I'll change this option to `True` then. However, both Autopep8 and Yapf are enabled too, so I don't know which one takes precedence.